lib/trino-filesystem-s3/src/test/java/io/trino/filesystem/s3/TestS3FileSystemS3Mock.java

```diff
@@ -32,7 +32,7 @@ public class TestS3FileSystemS3Mock
     private static final String BUCKET = "test-bucket";
 
     @Container
-    private static final S3MockContainer S3_MOCK = new S3MockContainer("3.0.1")
+    private static final S3MockContainer S3_MOCK = new S3MockContainer("3.10.0")
```

Comment:
set isCreateExclusive true here as well? or is it not yet supported?

Reply:
nvm just saw your comment in the description

Reply:
yeah, I'm gonna revert it. not needed.

Reply:
Reverted